cmd/sqlflowserver/main_test.go

```diff
@@ -123,8 +158,14 @@ func TestEnd2EndMySQL(t *testing.T) {
 	if testDBDriver != "mysql" {
 		t.Skip("Skipping mysql tests")
 	}
-	go start("mysql://root:root@tcp/?maxAllowedPacket=0")
+	dbStr := "mysql://root:root@tcp/?maxAllowedPacket=0"
+	go start(dbStr)
 	WaitPortReady("localhost"+port, 0)
```

Comment:
Not related to this PR...where is this variable defined in this go file?

Reply:
`port` is defined here: https://github.com/sql-machine-learning/sqlflow/blob/develop/cmd/sqlflowserver/main.go#L33